lib/smooth/application/display/ILI9341.cpp

```diff
@@ -204,43 +202,45 @@ namespace smooth::application::display
             // stack; we need this memory even when this function is finished because
             // the SPI driver needs access to it even while we're already calculating
             // the next line.
-            static std::array<spi_transaction_t, 6> trans;
+            static std::array<spi_transaction_t, 6> trans{};
```

Comment:
@enelson1001 I can't get the comment block below to make sense. `trans` is not on the stack, so we really shouldn't need to initialize it each time, right?

Reply:
Sorry for the delay I am in the USA and this week is Thanksgiving.
To be honest I just copied from esp-idf/examples/spi_master/main/spi_master_example_main.c without digging into the code.   I can see that some of the trans can be initialized once but x1, x2, y1,y2, data and pre and post trans states would still need to be set every time.  
1. Seems better to remove comment.  What would you like me to do?  
2. I updated my fork with git fetch upstream and did a git  checkout code-cleanup-SPI.  I tested the ili9341 with my M5Stack Demo 1 program and everything thing is still working.
3. I have some other questions but will wait until we get your current question resolved.

Reply:
Ok, I'll give it some thought. There is no rush, enjoy your stay.

Reply:
I ended up simply removing comment.